examples/btc-to-evm-fungible-transfer/README.md

```diff
@@ -0,0 +1,122 @@
+
+### 1. Clone the repository
+
+To get started, clone this repository to your local machine with:
+
+```bash
+git clone git@github.com:sygmaprotocol/sygma-sdk.git
+cd sygma-sdk/
+```
+
+### 2. Install dependencies
+
+Install the project dependencies by running:
+
+```bash
+yarn install
+```
+
+### 3. Build the sdk
+
+To start the example you need to build the sdk first with:
+
+```bash
+yarn build:all
+```
+
+## Usage
+
+This example uses the `dotenv` module to manage private keys and also to define env variables needed for this example to work. To run the example, you will need to configure your environment variables A `.env.sample` is provided as a template.
+
+**DO NOT COMMIT PRIVATE KEYS WITH REAL FUNDS TO GITHUB. DOING SO COULD RESULT IN COMPLETE LOSS OF YOUR FUNDS.**
+
+Create a `.env` file in the btc-to-evm example folder:
+
+```bash
+cd examples/btc-to-evm-fungible-transfer
+touch .env
+```
+
+Replace the values that are defined in the `.env.sample` file:
+
+```bash
+SYGMA_ENV=testnet
+BLOCKSTREAM_URL="your blockstream url"
+DESTINATION_ADDRESS="your evm destination address"
+DESTINATION_CHAIN_ID="destination domain chain id"
+RESOURCE_ID="resource id"
+SOURCE_CAIPID="source domain caip id"
+EXPLORER_URL="your bitcoin explorer url"
+MNEMONIC="your 12 or 24 mnemonic"
+DERIVATION_PATH="your derivation path"
+ADDRESS="your change address"
+AMOUNT="your amount to transfer"
+```
+
+* `DESTINATION_ADDRESS`: your `evm` destination address where you want your funds to be relayed
+* `DESTINATION_CHAIN_ID`: this is the chainId of the network where you want to receive the funds
+* `RESOURCE_ID`: the bitcoin resource id that can be found in our `shared-config` repository
+* `SOURCE_CAIPID`: caipId of the bitcoin domain
+* `MNEMONIC`: your testnet wallet mnemonic
+* `DERIVATION_PATH`: derivation path for your mnemonic. Use derivation path for either P2TR address or P2WPKH one
+* `ADDRESS`: the address from which we get the UTXO's and to which we send the change
+* `AMOUNT`: the actual amount to transfer
+
+Take into consideration that a typical response when query the utxos of your address look like this:
+
+```json
+{
+  "txid": "7bdf2ce472ee3c9cba6d2944b0ca6bcdceb4b893c7d2163678a0b688a8315d74",
+  "vout": 3,
+  "status": {
+      "confirmed": true,
+      "block_height": 2869535,
+      "block_hash": "",
+      "block_time": 1721666904
+  },
+  "value": 936396`
+}
+```
+
+Where `value` is the amount you have at your disposal and `vout` is the transaction output index.
+
+To send Testnet BTC to your EVM account on Sepolia using Taproot address run:
+
+```bash
+yarn run transfer:p2tr
+```
+
+To send Testnet BTC to your EVM account on Sepolia using P2WPKH address run:
+
+```bash
+yarn run transfer:p2wpkh
+```
+
+Replace the placeholder values in the `.env` file with your own Testnet BTC Taproot private key as well as the other env variables needed such as DESTINATION_ADDRESS, DESTINATION_DOMAIN_ID, RESOURCE_ID and SOURCE_DOMAIN_ID.
```

Comment:
is `DERIVATION_PATH` needed here?

Reply:
yes, for you to sign, is the same as we use private key in the other examples